src/ResourceManagement/Scheduler/SchedulerManagement/packages.config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.Rest.ClientRuntime" version="1.0.2" targetFramework="net45" />
```

Comment:
Please use Microsoft.Rest.ClientRuntime 1.2.0 and Microsoft.Rest.ClientRuntime.Azure 1.0.20